djstripe/exceptions.py

```diff
@@ -1,14 +1,28 @@
 # -*- coding: utf-8 -*-
+"""
+.. module:: djstripe.exceptions.
+
+   :synopsis: dj-stripe Exceptions.
+
+.. moduleauthor:: @kavdev
+
+"""
 from __future__ import unicode_literals
 
 
 class MultipleSubscriptionException(Exception):
+    """Raised when Customer has multiple Subscriptions."""
```

Comment:
"""Raised when a Customer has multiple Subscriptions and only one is expected."""